### PR TITLE
docs: Add section about stale resources to FAQ

### PR DESCRIPTION
**Pull request by shimonp21**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
-No. CloudQuery only access meta-data and configuration data. It never pulls data from your application databases or cloud storage files.
+No. CloudQuery only accesses metadata and configuration data. It never pulls data from your application databases or cloud storage files.
+
+### What happens when I run two (or more) fetches? Will the second fetch remove resources that no longer exist from the database?
+
+After a `fetch`, your PostgreSQL database will always mirror your cloud environment. This means that a `fetch` will add to the database
+any resources that were added to the cloud during the time from the previous `fetch`, update any resources that were changed in the cloud, 
+and remove from the database any resources that were removed from the cloud.
+
+The only caveat is that in the case of configuration changes (e.g. an AWS account was removed from `cloudquery.yml`),
```
Comment (bbernays) [inline docs/faq.md:13]: Maybe make this clear that this applies to the following scenarios:
1. Resource removed from list of `resources`
2. When using AWS Orgs if an account is deleted or removed from the org then the meta data about that resource is left untouched
3. When finding GCP Projects via Folders/Org queries then projects that no longer return in the query will be left alone

Comment (roneli) [reply docs/faq.md:13]: Will add my 2cents here (adding to what ben already said which is correct)
1. if you fetch same account, but remove a resource from the list, that resource will remain untouched and won't be deleted / updated so it won't "mirror" the cloud
2. Like ben said in 2, basically only accounts fetched will get updated, to remove accounts that were not updated recently, you should use the purge command.

File: docs/faq.md [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
+### What happens when I run two (or more) fetches? Will the second fetch remove resources that no longer exist from the database?
+
+After a `fetch`, your PostgreSQL database will always mirror your cloud environment. This means that a `fetch` will add to the database
+any resources that were added to the cloud during the time from the previous `fetch`, update any resources that were changed in the cloud,
+and remove from the database any resources that were removed from the cloud.
+
+The only caveat is that in the case of configuration changes, subsequent fetches won't always remove stale resources. For instance:
+- If an account was removed from `cloudquery.yml`, subsequent fetches won't process that account (and won't remove that account's resources).
+- If a resource type was removed from `cloudquery.yml`, that resource type won't get fetched (and database entries for this resources also won't get removed).
+- When using AWS orgs, if an account is deleted or removed from the org then the database entries for that account will be untouched.
```
Comment (bbernays) [inline docs/faq.md:16]:
```suggestion
- When using [AWS Orgs](https://hub.cloudquery.io/providers/cloudquery/aws/latest#aws-organizations), if an account is deleted or removed from the org then the database entries for that account will be untouched.
```

File: docs/faq.md [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+- When using GCP folders, if a project is deleted or removed from the folder, the database entries for that account will be untouched.
```
Comment (bbernays) [inline docs/faq.md:17]:
```suggestion
- When using [GCP Folders or Organizations](https://hub.cloudquery.io/providers/cloudquery/gcp/latest#configuration), if a project is deleted or removed from the folder, the database entries for that account will be untouched.
```

**Review by bbernays — APPROVED** [review]

**Pull request merged**